bgpd/bgp_vty.c

```diff
@@ -10813,6 +10859,36 @@ static void bgp_show_peer(struct vty *vty, struct peer *p, bool use_json,
 							"received");
 				}
 
+				/* BGP Extended Message */
+				if (CHECK_FLAG(p->cap, PEER_CAP_EXT_MESSAGE_RCV)
```

Comment:
I really don't like doing a check on both, then doing a check on each, especially given how compressed the code is already.